doc/release-notes.md

```diff
@@ -28,111 +28,22 @@ reindex or re-sync the whole chain.
 
 # Notable changes
 
-## HD Wallets Enabled by Default
+## Work Queue RPC Fix / Deadlock Fix
 
-In this release, we are taking a significant step towards enhancing the Dash wallet's usability by enabling Hierarchical
-Deterministic (HD) wallets by default. This change aligns the behavior of `dashd` and `dash-qt` with the previously
-optional `-usehd=1` flag, making HD wallets the standard for all users.
+A deadlock causing nodes to become non-responsive and RPC to report "Work depth queue exceeded".
+Thanks to @knst who discovered the cause. This previously caused masternodes to become PoSe banned.
```

Comment:
We used to avoid `@` mentions in release notes (not 100% sure why though :D) so maybe
```suggestion
Thanks to Konstantin Akimov (knst) who discovered the cause. This previously caused masternodes to become PoSe banned.
```

Reply:
Dashwallet